pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -59,16 +59,24 @@ func (tc *TidbCluster) PDAutoFailovering() bool {
 	return false
 }
 
-func (tc *TidbCluster) PDRealReplicas() int32 {
+func (tc *TidbCluster) PDDesiredReplicas() int32 {
 	return tc.Spec.PD.Replicas + int32(len(tc.Status.PD.FailureMembers))
 }
 
+func (tc *TidbCluster) PDActualReplicas() int32 {
```

Comment:
We use sts to manage pods, so the status of the replica of sts is actual replicas.